src/i18n/en-US.json

```diff
@@ -1166,7 +1166,7 @@
   "ongoingGroupAudioCall": "Ongoing conference call with {{conversationName}}.",
   "ongoingGroupVideoCall": "Ongoing video conference call with {{conversationName}}, your camera is {{cameraStatus}}.",
   "ongoingVideoCall": "Ongoing video call with {{conversationName}}, your camera is {{cameraStatus}}.",
-  "otherUserNotSupportMLSMsg": "You can't communicate with [bold]{{participantName}}[/bold] anymore, as you two now use different protocols. When [bold]{{participantName}}[/bold] gets an update, you can call and send messages and files again.",
+  "otherUserNotSupportMLSMsg": "You can't communicate with {{participantName}} anymore, as you two now use different protocols. When {{participantName}} gets an update, you can call and send messages and files again.",
```

Comment:
Question: Is localisation in German done for those? Might be worth it to cherrypick the localisation commit too if we want to avoid the `[bold]` tag to appear as a string

Reply:
When merged to dev, these changes were made automatically for German

Reply:
So I guess it will happen automatically here as well

Reply:
Let's see, it didn't work out that way on master last time :| 
Let's worry about it when we get there :crossed_fingers: